src/Rules/SequentialCharacters.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace LangleyFoxall\LaravelNISTPasswordRules\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+
+/**
+ * Class SequentialCharacters.
+ *
+ * Implements the 'Sequential characters' recommendation
+ * from NIST SP 800-63B section 5.1.1.2.
+ */
+class SequentialCharacters implements Rule
+{
+
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param string $attribute
+     * @param mixed  $value
+     *
+     * @return bool
+     */
+    public function passes($attribute, $value)
```

Comment:
This is good for an initial rule. I think [my suggestion](https://github.com/langleyfoxall/laravel-nist-password-rules/issues/16#issuecomment-507964062) would be more performant.

We can always improve and tweak things later on down the line. Let's get this in for now 👍

Reply:
@dextermb oh sorry I hadn't seen this before commit and submitting a PR I can implement this now if we want? I'm happy to do that.

and @DivineOmega I've gone ahead and made the change you requested.

Reply:
> @dextermb oh sorry I hadn't seen this before commit and submitting a PR I can implement this now if we want?

Either is fine 👍 I'm sure stuff will be tweaked as the package continues to be used either way.

Reply:
Let's get this merged as is. We can tweak it later on if necessary.